samples/csharp_dotnetcore/14.nlp-with-dispatch/NlpDispatch/NlpDispatchBot.cs

```diff
@@ -27,23 +27,23 @@ public class NlpDispatchBot : IBot
         /// <summary>
         /// Key in the Bot config (.bot file) for the Home Automation Luis instance.
         /// </summary>
-        public static readonly string HomeAutomationLuisKey = "Home Automation";
+        public const string HomeAutomationLuisKey = "Home Automation";
 
         /// <summary>
         /// Key in the Bot config (.bot file) for the Weather Luis instance.
         /// </summary>
-        public static readonly string WeatherLuisKey = "Weather";
+        public const string WeatherLuisKey = "Weather";
 
         /// <summary>
         /// Key in the Bot config (.bot file) for the Dispatch.
         /// </summary>
-        public static readonly string DispatchKey = "nlp-with-dispatchDispatch";
+        public const string DispatchKey = "nlp-with-dispatchDispatch";
 
         /// <summary>
         /// Key in the Bot config (.bot file) for the QnaMaker instance.
         /// In the .bot file, multiple instances of QnaMaker can be configured.
         /// </summary>
-        public static readonly string QnAMakerKey = "sample-qna";
+        public const string QnAMakerKey = "sample-qna";
```

Comment:
I think you _could_ in this case change these to const, however, they _should_ be private as far as I can see.
And as a general rule favor static readonly. But if this was private then I think we don't care too much.